### PR TITLE
[DRAFT] Add service properties to BELLs

### PR DESCRIPTION
**Pull request by dazavala**
Add service properties to BELLs

### CODE REVIEW

**Comment by skasund** [comment]
L2 message review completed

File: dev/com.ibm.ws.classloading.bells/resources/OSGI-INF/l10n/bells.properties [hunk 1]
```diff
@@ -23,3 +23,18 @@ bell.library.ref.desc=The library to use for the BELL.
 
 service.name=Service type
 service.name.desc=The name of the service that the system will look up in the /META-INF/services folder. If not specified, the system discovers all services located in the META-INF/services folder.
+
+service.properties=Service properties
+service.properties.desc=Properties the system will pass to a service implementation at creation. The service implementation class must declare a public constructor with one parameter of type java.util.Map<String, String>.
```
Comment (helyarp) [inline dev/com.ibm.ws.classloading.bells/resources/OSGI-INF/l10n/bells.properties:28]: Change "will pass" to "passes"

File: dev/com.ibm.ws.classloading.bells/resources/OSGI-INF/l10n/bells.properties [hunk 2]
```diff
@@ -23,3 +23,18 @@ bell.library.ref.desc=The library to use for the BELL.
...
+
+service.properties.service.name=Service type
+service.properties.service.name.desc=The name of the service in the /META-INF/service folder to which the properties apply.  If not specified, the properties apply to all services.
+
+service.properties.property=Name/value pair describing a service property
+service.properties.property.desc=A property useful for service configuration. A service property must have a non-empty name and value, and all property names must be unique. The system will ignore invalid services properties.
```
Comment (helyarp) [inline dev/com.ibm.ws.classloading.bells/resources/OSGI-INF/l10n/bells.properties:34]: Change "will ignore" to "ignores"

**Review by helyarp — COMMENTED** [review]
Suggested changes

**Comment by dazavala** [comment]
This is the prototype.  Closing now as the socialized version of this enhancement recently delivered via [PR 21529](https://github.com/OpenLiberty/open-liberty/pull/21529).

**Pull request closed without merging**